solidity/external/mainnet/TBTCToken.json

```diff
@@ -0,0 +1,401 @@
+{
+  "address": "0x8dAEBADE922dF735c38C80C7eBD708Af50815fAa",
+  "abi": [
```

Comment:
Do we need "abi" here or just an address?

Reply:
For now, it's just the address but I think a good practice is to have an abi as well (if possible). We may need it in the future. This is also how the `TBTCToken` dependency is handled in coverage pools.